### PR TITLE
Combine consecutive issets

### PR DESCRIPTION
**Pull request by MathiasReker**

### CODE REVIEW

File: classes/TaskRunner/Miscellaneous/UpdateConfig.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function run()
             $config['channel'] = $request['channel'];
             $config['archive.filename'] = Upgrader::DEFAULT_FILENAME;
         }
-        if (isset($request['private_release_link']) && isset($request['private_release_md5'])) {
+        if (isset($request['private_release_link'], $request['private_release_md5'])  ) {
```
Comment (PierreRambaud) [inline classes/TaskRunner/Miscellaneous/UpdateConfig.php]: You add useless spaces :)

Comment (MathiasReker) [reply classes/TaskRunner/Miscellaneous/UpdateConfig.php]: Thank you @PierreRambaud. I fixed this part in my script. :)

File: classes/TaskRunner/Upgrade/BackupDb.php [hunk 2]
```diff
@@ -159,8 +159,8 @@ public function run()
                 $schema = $this->container->getDb()->executeS('SHOW CREATE TABLE `' . $table . '`', true, false);
 
                 if (count($schema) != 1 ||
-                    !((isset($schema[0]['Table']) && isset($schema[0]['Create Table']))
-                        || (isset($schema[0]['View']) && isset($schema[0]['Create View'])))) {
+                    !((isset($schema[0]['Table'], $schema[0]['Create Table']))
```
Comment (PierreRambaud) [inline classes/TaskRunner/Upgrade/BackupDb.php]: You can also remove useless parenthesis :D 
(I know you hate me :roll_eyes: )

Comment (MathiasReker) [reply classes/TaskRunner/Upgrade/BackupDb.php]: Fixed ;) Thanks

**Pull request merged**